modules/restore-replace-stopped-baremetal-etcd-member.adoc

```diff
@@ -0,0 +1,579 @@
+// Module included in the following assemblies:
+//
+// * /backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.adoc
+
+:_content-type: PROCEDURE
+[id="restore-replace-stopped-baremetal-etcd-member_{context}"]
+= Replacing an unhealthy bare metal etcd member whose machine is not running or whose node is not ready
+
+This procedure details the steps to replace a bare metal etcd member that is unhealthy either because the machine is not running or because the node is not ready.
+
+If you are running installer-provisioned infrastructure or you used the Machine API to create your machines, follow these steps. Otherwise you must create the new master using the same method that was used to originally create it. 
+
+.Prerequisites
+
+* You have identified the unhealthy bare metal etcd member.
+* You have verified that either the machine is not running or the node is not ready.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have taken an etcd backup.
++
+[IMPORTANT]
+====
+It is important to take an etcd backup before performing this procedure so that your cluster can be restored if you encounter any issues.
+====
+
+.Procedure
+
+. Verify and remove the unhealthy member.
+
+.. Choose a pod that is _not_ on the affected node:
++
+In a terminal that has access to the cluster as a `cluster-admin` user, run the following command:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-etcd -o wide | grep etcd | grep -v guard
+----
++
+.Example output
+[source,terminal]
+----
+etcd-openshift-master-10                5/5     Running     11          3h56m   192.168.10.9   openshift-master-10   <none>           <none>
+etcd-openshift-master-11                5/5     Running     0          3h54m   192.168.10.10   openshift-master-11   <none>           <none>
+etcd-openshift-master-12                5/5     Running     0          3h58m   192.168.10.11   openshift-master-12   <none>           <none>
```

Comment:
why 10, 11, 12?

Reply:
The prior ones were in sequence, 0, 1, 2. I wanted to change the text somewhat, so to keep the sequence "correct", I chose simply to add 10 to each name.